internal/cephfs/errors.go

```diff
@@ -44,3 +52,7 @@ type ErrVolumeNotFound struct {
 func (e ErrVolumeNotFound) Error() string {
 	return e.err.Error()
 }
+
+func (e ErrVolumeNotFound) Unwrap() error {
```

Comment:
Need to add a comment for exported functions

Reply:
yeah, but it was not done for others, so I skipped it here. Will add comments for the other ones as well.